metadata/service/server/local.yml

```diff
@@ -9,8 +9,5 @@ parameters:
       bind:
         address: 127.0.0.1
         port: 9200
-      index:
-        shards: 1
-        replicas: 0
```

Comment:
In my idea this is set at the "model" level because you know your infra but that is true that local and single have special meaning. I will change it.